charts/atlas-cluster/values.yaml

```diff
@@ -0,0 +1,62 @@
+mongodb-atlas-operator:
+  enabled: true
+
+# Please provide Atlas API credentials and Organization
+atlas:
+  orgId: "<Atlas Organization ID>"
+  publicApiKey: "<Atlas_api_public_key>"
+  privateApiKey: "<Atlas_api_private_key>"
+
+  connectionSecretName: ""
+
+project:
+  enable: true
+  name: my-project
+  atlasProjectName: "AndreyB"
+
+  projectIpAccessList:
+    - ipAddress: "192.0.2.15"
+      comment: "IP address for Application Server A"
+    - ipAddress: "203.0.113.0/24"
+      comment: "CIDR block for Application Server B - D"
+
+mongodb:
+  providerSettings:
+    instanceSizeName: M10
+    providerName: AWS
+    regionName: US_EAST_1
+
+# Optional section. for more information visit
+# https://docs.atlas.mongodb.com/reference/api/clusters-create-one/
+# #
+#   autoScaling:
+#     autoIndexingEnabled: false
+#     compute:
+#       enabled: false
+#       maxInstanceSize: 'M40'
+#       minInstanceSize: 'M10'
+#       scaleDownEnabled: false
+#     diskGBEnabled: false
+#   biConnector:
+#     enabled: false
+#     readPreference: 'primary'
+#   clusterType: 'REPLICASET'
+#   diskSizeGB: '50'
+#   encryptionAtRestProvider: 'AWS'
+#   labels:
+#   mongoDBMajorVersion: '4.4'
+#   numShards: 2
+#   paused: false
+#   pitEnabled: false
+#   providerBackupEnabled: false
+#   replicationSpecs:
+users:
```

Comment:
Good idea having the list of users.

maybe we could use the same idea for clusters? So the list of clusters for the project?
In this case it may make sense to rename the helm chart to `atlas-project`. This would allow to create more than one cluster in one project and support it easier. So just editing the values.yaml file would be necessary - adding the users/clusters (whatever in future).

Reply:
Good points here.
I think there is a case for a "quick start" chart which is Project + 1 Cluster, then more general charts for the more advanced scenarios.

Reply:
that would make HELM chart much more complicated. 
I would like this HELM chart to equate with a single cluster. (KISS)
Maybe we could have a "fleet" HELM chart that deploys multiple clusters

Reply:
After looking at the proposed atlas-cluster chart, it seems flexible enough since you can run it once, then again with project=No. 
It also seems to be a good logical fit to map a cluster with a "helm release". The "fleet" is then running chart multiple times script flip in the project enable bit.

Reply:
we discussed this with Andrey and decided to keep to the paradigm one cluster per project. In this case it will be safe to create another cluster with a different name without passing `enable` flags. Both users and projects include the cluster name in their names so `helm install c1` and the next `helm install c2` will work correctly